packages/config/config/devices/0x027a/zen26_1.0-1.0.json

```diff
@@ -50,37 +61,59 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "LED on when switch off",
+					"label": "LED is on when switch is off",
 					"value": 0
 				},
 				{
-					"label": "LED on when switch on",
+					"label": "LED is on when switch is off",
 					"value": 1
 				},
 				{
-					"label": "LED off",
+					"label": "LED is always off",
 					"value": 2
 				},
 				{
-					"label": "LED on",
+					"label": "LED is always on",
 					"value": 3
 				}
 			]
 		},
 		"3": {
 			"label": "Auto Turn-Off Timer",
-			"description": "enable the Auto Turn-Off Timer",
-			"valueSize": 2,
+			"description": "Enable the auto turn-off timer.",
+			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 32767,
+			"maxValue": 1,
```

Comment:
I wasn't sure for version 1.0. For the later firmware revisions, they changed the unit to minutes and increased the max to 65535 and increased the field size to 4 bytes. https://www.support.getzooz.com/kb/article/315-zen26-s2-on-off-switch-ver-3-02-advanced-settings/ For version 1.0 I wasn't sure, but I'm going to assume the manual is correct since it also uses seconds as the units. I'll fix this, and also double check the valueSize on the rest of the files.

Reply:
A little background: Z-Wave config parameters are meant to be signed by default (unless a device interprets them differently, which is why we have the `unsigned` flag). So you'd have a range of -32768..+32767. I've seen enough files where +32768 (or similar for 1 und 4-byte values) was used on accident, so I'm careful around this stuff.

Reply:
Actually, after reading the manual, I'm even more confused. The manual says the timer is controlled entirely in parameter 3, but the docs have parameter 3 as enabled/disabled, and parameter 4 as the length. I'm going to reach out to Zooz support on this one to see if they still have documentation for version 1.0 of the firmware available.

Reply:
I'm still waiting to hear back from Zooz support, but for now, I'm going to go under the assumption that the manuals for the 26 and 27 correctly describe the config parameters for the 1.0 firmware for each device. The copyright year on the manuals do seem to align with the firmware release date. For newer firmware revisions, I'm trusting the documentation on their website or my own testing where I can test.